### PR TITLE
vscode.d.ts: clarified what priority means for status items

### PR DESCRIPTION
**Pull request by johnpapa**

### CODE REVIEW

File: src/vs/vscode.d.ts [hunk 1]
```diff
@@ -4556,8 +4556,8 @@ declare module 'vscode' {
 		readonly alignment: StatusBarAlignment;
 
 		/**
-		 * The priority of this item. Higher value means the item should
-		 * be shown more to the left.
+		 * The priority of this item. Higher value (the lower the number the higher the priority) 
+		 * means the item should be shown more to the left.
 		 */
```
Comment (gregvanl) [inline src/vs/vscode.d.ts]: Still somewhat confusing. How about "The priority of this item. Lower numerical values indicate the item should be shown more to the left."

**Comment by jrieken** [comment]
ping @johnpapa for the feedback, ping @bpasero for the change

**Review by bpasero — CHANGES_REQUESTED** [review]
The doc change is not correct. This priority value translates 1 to 1 to the [internal priority value](https://github.com/Microsoft/vscode/blob/8ffe01a66584ca8c6d2ece05df0d1f4c7fe5c2a6/src/vs/platform/statusbar/common/statusbar.ts#L71) and that is documented as: `items get arranged from highest priority to lowest priority from left to right in their respective alignment slot`

**Comment by bpasero** [comment]
I am going ahead and close this as I think the current doc actually reflects this.

**Pull request closed without merging**